packaging/buildozer.spec

```diff
@@ -258,9 +329,12 @@ android.allow_backup = True
 #p4a.fork = kivy
 
 # (str) python-for-android branch to use, defaults to master
-#p4a.branch = master
+p4a.branch = release-2022.12.20
```

Comment:
Any reason for pinning this exact version and not `master` ?

Reply:
Yes, as it happens! See comment below.